collectors/ebpf.plugin/ebpf.d/oomkill.conf

```diff
@@ -0,0 +1,7 @@
+# The `ebpf load mode` option accepts the following values :
+#  `entry` : The eBPF collector only monitors calls for the functions, and does not show charts related to errors.
+#  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
+#            new charts for the return of these functions, such as errors.
+[global]
+    ebpf load mode = entry
+    update every = 1
```

Comment:
When there is integration with `apps`, this file needs to have more line, please, take a look on `vfs.conf` for the missing lines.

Reply:
In this case we actually always need apps integration, because that's the only chart we create - the one that requires apps group information.

Reply:
So in other words, disabling apps integration here would be equivalent to disabling the whole thing.

Reply:
All right, I understood, but this is not explicit in this file and I also did not see any information on documentation related to this chart, please, can you add them for us?

Reply:
Added in https://github.com/netdata/netdata/pull/11470/commits/d9fc39769bbf9493a7c1c8dc3a6bc3fce2728869